UI/SidePanel.cpp

```diff
@@ -1306,6 +1344,50 @@ std::set<TemporaryPtr<const Ship> > AutomaticallyChosenInvasionShips(int target_
     return retval;
 }
 
+/** Returns valid Ship%s capable of bombarding a given planet.
```

Comment:
Allows doxygen to link the `Ship` class here, with the appended `s`.
e.g. XMLElement::WriteElement
[code](https://github.com/freeorion/freeorion/blob/master/util/XMLDoc.h#L329)
[doc](http://freeorion.github.io/cppapi/master/classXMLElement.html#a486644805342c20a02163443d35c553a)